observable-slim.js

```diff
@@ -500,7 +526,31 @@ var ObservableSlim = (function() {
 			if (foundMatch === true) {
 				observables.splice(c,1);
 			}
-		}
+		},
+
+        /*	Method: beforeChange
+        		This method accepts a function will be invoked before changes.
+
+			Parameters:
+				proxy 	- the ES6 Proxy returned by the create() method.
+				callback 	- Function, will be invoked before every change is made to the proxy, if it returns false no changes will be made.
+		*/
+        beforeChange: function (proxy, callback) {
```

Comment:
This looks good to me, but it has the same issue with the mixing of tabs and spaces. It might take one more re-do of this pull request to resolve the issue.